recipes-connectivity/prplmesh/prplmesh.bb

```diff
@@ -45,12 +46,22 @@ BEEROCKS_WLAN1_IFACE_raspberrypi-rdk-broadband = "wlan0"
 BEEROCKS_WLAN2_IFACE_raspberrypi-rdk-broadband = "wlan1"
 BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE_raspberrypi-rdk-broadband="/var/run/hostapd0/wlan0"
 BEEROCKS_HOSTAP_WLAN2_CTRL_IFACE_raspberrypi-rdk-broadband="/var/run/hostapd4/wlan1"
-BEEROCKS_BH_WIRE_IFACE_raspberrypi-rdk-broadband = "erouter0"
+#
+# Turris Omnia machine specific
+#
+BEEROCKS_WLAN1_IFACE_turris = "wifi0"
+BEEROCKS_WLAN2_IFACE_turris = "wifi1"
+BEEROCKS_WLAN3_IFACE_turris = "wifi2"
+
+BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE="/var/run/hostapd/wifi0"
```

Comment:
Looks like we just overwrite `BEEROCKS_HOSTAP_WLAN1_CTRL_IFACE` and it can cause the problems in other platforms...